.circleci/config.yml

```diff
@@ -40,8 +40,14 @@ commands:
           key: *hunter-cache-key
           paths:
             - ~/.hunter
+      # Build the silkrpc folder first so the generated files are available for the other folders
       - run:
           name: "Build"
+          working_directory: ~/build
+          command: cmake --build ~/build/silkrpc
```

Comment:
Is this necessary for the generation of additional `proto` binding files? If so, please see my comments below